README.md

```diff
@@ -311,6 +311,10 @@ libraries (.so). The gcc provided by Developer Toolset and rh-python36 should be
 Also, the libstdc++ has to be linked statically to avoid discrepancy of libstdc++ installed on
 CentOS vs. newer gcc version by devtoolset.
 
+Furthermore, a special flag `--//tensorflow_io/core:static_build` has to be passed to Bazel
```

Comment:
@yongtang not sure if it is the right way but can we use `plugin_build` instead of `static_build`?

Reply:
@kvignesh1420 The reason to add this special flag is a little tricky.

In this PR, we build two shared libraries, one is `libtensorflow_io.so` which depends on C++ API and will not be usable for later versions of TF (e.g., 2.5, 2.6, etc as we only build against TF 2.4).

The other shared library is the newly introduced `libtensorflow_io_plugins.so` which depends on C API only and will be usable for later versions of TF (it can be used for TF 2.6, 2.7, etc). 

However, the only glitch is the CentOS 7 platform. On CentOS 7 we can only use RedHat's Software Collections to get a newer version of gcc (default versions of gcc on CentOS 7 will not work), and with static C++ linkage. Because of that, having two shared libraries `libtensorflow_io_plugins.so` and `libtensorflow_io.so` will not work and will cause segment fault.

As a result, we actually merges `libtensorflow_io_plugins.so` as part of `libtensorflow_io.so`. That is why we call it `static_build` to indicating it is for C++ static linkage.